cMake/FreeCAD_Helpers/SetupLibYaml.cmake

```diff
@@ -0,0 +1,11 @@
+macro(SetupYamlCpp)
+# -------------------------------- YamlCpp --------------------------------
+
+    find_package(yaml-cpp REQUIRED)
```

Comment:
Marking this as "REQUIRED" should eliminate the need for the test code after it, right?